### PR TITLE
added aszalea

### PR DESCRIPTION
**Pull request by rboeglinjr2010**
hey i did a thing and its better. commit plz

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,37 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <link rel="stylesheet" href="style.css"/>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (rboeglinjr2010) [inline index.html:6]: put that back

File: index.html [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+    <a
+      href="https://hb.imgix.net/cfc319ad11b0b75902e626c54b0b5719f90e78c3.png?auto=compress,format&fit=crop&h=353&w=616&s=7b59ce6671b794b283271f19ed51b6b9"
+      target="blank"
+      ><button>click me</button></a
+    >
     <ol>
-        <li>milk</li>
-        <li>eggs</li>
+      <li>milk</li>
+      <li>eggs</li>
```
Comment (rboeglinjr2010) [inline index.html:42]: looks better but make it larger

**Review by rboeglinjr2010 — COMMENTED** [review]
looks pretty good. see comments

File: index.html (hunk 1 above)
Comment (rboeglinjr2010) [reply index.html:6]: fixed

Comment (Aszalea-Calderon) [reply index.html:6]: fixed it

**Pull request merged**